.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+name: Docurium
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [head, 2.6, 2.5, 2.4]
+        os: [ ubuntu-18.04, macos-latest ]
+
+    name: Ruby ${{ matrix.ruby }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ endsWith(matrix.ruby, 'head') || matrix.ruby == 'debug' }}
```

Comment:
Is "debug" left over from some other testing? Is the `endsWith` necessary? Wouldn't `matrix.ruby` match `head` exactly?

Reply:
That's a verbatim copy-paste from the documentation, where there's much more than MRI Ruby handled. I'm tempted to keep it as-is for future-proofing, but I'll gladly defer to your preference.